server/web/web.go

```diff
@@ -72,19 +72,19 @@ func New() (*gin.Engine, error) {
 }
 
 func handleCustomFilesAndAssets(fs *prefixFS) func(ctx *gin.Context) {
-	serveFileOrEmptyContent := func(w http.ResponseWriter, r *http.Request, localFileName string) {
+	serveFileOrEmptyContent := func(w http.ResponseWriter, r *http.Request, localFileName, contentTypeName string) {
```

Comment:
contentTypeName not really what is passed there as file name is being passed to this param

Reply:
thanks for the feedback.
well, the variable name might not be named optimal.
According to the docs, `http.ServeContent(..., name, ...)` just using the name to determine the content type.

Any proposal on how to improve the name, @lafriks ?

Reply:
I would probably call it `fileName`